modules/openapi-generator/src/main/resources/cpp-pistache-server/api-header.mustache

```diff
@@ -14,24 +14,21 @@
 #include <pistache/http_headers.h>
 #include <pistache/optional.h>
 {{^hasModelImport}}#include <nlohmann/json.hpp>{{/hasModelImport}}
+#include <utility>
 
 {{#imports}}{{{import}}}
 {{/imports}}
 
-{{#apiNamespaceDeclarations}}
-namespace {{this}} {
-{{/apiNamespaceDeclarations}}
-
-{{#hasModelImport}}
-using namespace {{modelNamespace}};{{/hasModelImport}}
+namespace {{apiNamespace}}
```

Comment:
This will only work with C++17 or newer, while Pistache is guaranteed to work with C++14, so I'd return to older syntax. This comment applies to all namespace definition changes.

Reply:
Pistache merged switch C++17 2 hours after my comment, so you can ignore this comment.